frontends/systolic-lang/gen_post_op.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python3
+
+import calyx.builder as cb
+from calyx import py_ast
+from gen_array_component import NAME_SCHEME
+from gen_pe import BITWIDTH, INTWIDTH, FRACWIDTH
+from fud.stages.verilator import numeric_types
```

Comment:
Ah, weird that the numeric types stuff still resides in the `verilator` stage. Can you open an issue saying that it should be moved to `calyx-py` instead?